kornia/feature/__init__.py

```diff
@@ -23,5 +39,13 @@
     "ellipse_to_laf",
     "make_upright",
     "get_laf_scale",
-    "SIFTDescriptor"
+    "scale_laf",
+    "SIFTDescriptor",
+    "PassLAF",
+    "ScaleSpaceDetector",
+    "LAFAffineShapeEstimator",
+    "PatchAffineShapeEstimator",
+    "LAFOrienter",
+    "PatchDominantGradientOrientation",
+    "raise_error_if_laf_is_not_valid"
```

Comment:
do we want to expose this or it's just for internal use ?

Reply:
I think, it is useful enough, if users would like to do their own checks. But don`t have a strong opinion on this

Reply:
maybe yes, but I would add it to the `check_` functions we were discussing in another thread